### ISSUE TITLE
Publish to Packagist.org?

### ISSUE THREAD

**Issue by danielbachhuber** [jaredatch/EA-Share-Count#103]
👋  I'd like to be able to `composer require` this plugin into a site. However, `composer require` against a bare Git repo is less than desirable.

Would you be open to publishing to [Packagist](https://packagist.org/)?

Thanks!

**Comment by billerickson** [comment]
EA Share Count has been replaced by [Shared Counts](https://github.com/jaredatch/shared-counts). I recommend you switch any sites that are using the old EA Share Count over to Shared Counts.

I'd be interested in adding Shared Counts to Packagist but have never done it before. Is there something we need to add to the plugin to make it work, or just submit it to the directory?

**Comment by danielbachhuber** [comment]
> EA Share Count has been replaced by [Shared Counts](https://github.com/jaredatch/shared-counts). I recommend you switch any sites that are using the old EA Share Count over to Shared Counts.

Hm. I wonder if we're even using it then...

> I'd be interested in adding Shared Counts to Packagist but have never done it before. Is there something we need to add to the plugin to make it work, or just submit it to the directory?

If you already have a `composer.json` file, it's as easy as submitting it to the directory.

**Comment by danielbachhuber** [comment]
> > EA Share Count has been replaced by [Shared Counts](https://github.com/jaredatch/shared-counts). I recommend you switch any sites that are using the old EA Share Count over to Shared Counts.
> 
> Hm. I wonder if we're even using it then...

Yeah, it looks like we're using it solely to fetch Pinterest share counts
